src/build.ts

```diff
@@ -304,7 +320,7 @@ async function accumulateSize(
     const fileResolution = resolveFile(file);
     if (isAssetPath(fileResolution)) {
       try {
-        size += (await stat(resolveOutput(fileResolution))).size;
+        size += (await maybeStat(resolveOutput(fileResolution)))?.size ?? 0;
```

Comment:
Same question as above — this seems like an unrelated bug fix/change and would appreciate context?